portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,177 @@
 <!DOCTYPE html>
 <html>
-  <head>
+    <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cynthia HU</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
+    </head>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+        <h1>Cynthia HU</h1>
+        <ul class="navbar">
+            <li><a href="#about">About</a></li>
+            <li><a href="#tech-corner">Tech Corner</a></li>
+            <li><a href="#travel-log">Travel Log</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+        <div id="about">
+            <h2>About</h2>
+           
+            <div class="image-box">
+                <img src="/images/profile3.jpg" alt="me in Hokkaido, Japan" class="image" width="560px" height="420px" align="middle">
```

Comment:
If you change these to "./images/profile3.jpg" then it will work in the HTML preview.